en-US/Players_Guide/03_Creating_a_Character/Classes/Vanguard/Vanguard.md

```diff
@@ -1,8 +1,236 @@
 #### Vanguard
 
-> **Warning:**
-> This class has not yet been transferred from the Google Doc.
+Vanguards channel divine strength, standing as unyielding champions of causes greater than themselves.
 
+> **Breaking your Oath**
```

Comment:
I moved this up from the end of the class, since it's a sidebar about the whole class rather than about a subclass feature (where it would lie structurally, if left there).